ui/hooks/useTokensToSearch.js

```diff
@@ -40,28 +40,32 @@ export function getRenderableTokenData(
   useTokenDetection,
 ) {
   const { symbol, name, address, iconUrl, string, balance, decimals } = token;
+  let contractExchangeRate;
+  if (isSwapsDefaultTokenSymbol(symbol, chainId)) {
+    contractExchangeRate = 1;
+  } else if (string && conversionRate > 0) {
```

Comment:
Why do we need to check for the "string" value here?

Reply:
Because `string` is also passed into the `getTokenFiatAmount` fn as `tokenAmount` and if it's undefined, then it exits early. Basically, we only want to do the expensive `toChecksumHexAddress` call if it will be used later on.

Reply:
Ah ok. Maybe worth adding a comment about that here, but I won't block on it!